apps/condo/domains/settings/schema/MobileFeatureConfig.test.js

```diff
@@ -0,0 +1,217 @@
+                    expect(obj.dv).toEqual(1)
+                    expect(obj.sender).toEqual(attrs.sender)
+                    expect(obj.createdBy).toEqual(expect.objectContaining({ id: client.user.id }))
+                })
+
+                test('can update', async () => {
+                    const client = await makeClientWithNewRegisteredAndLoggedInUser()
+                    const [organization] = await createTestOrganization(admin)
+
+                    const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                        canManageMobileFeatureConfigs: true,
+                    })
+                    await createTestOrganizationEmployee(admin, organization, client.user, role)
+
+                    const [objCreated] = await createTestMobileFeatureConfig(admin, organization)
+
+                    const [obj, attrs] = await updateTestMobileFeatureConfig(client, objCreated.id)
+
+                    expect(obj.id).toMatch(UUID_RE)
+                    expect(obj.dv).toEqual(1)
+                    expect(obj.sender).toEqual(attrs.sender)
+                    expect(obj.v).toEqual(2)
+                    expect(obj.updatedBy).toEqual(expect.objectContaining({ id: client.user.id }))
+                })
+
+                test('can\'t delete', async () => {
+                    const client = await makeClientWithNewRegisteredAndLoggedInUser()
+
+                    await expectToThrowAccessDeniedErrorToObj(async () => {
+                        await MobileFeatureConfig.delete(client, 'id')
+                    })
+                })
+
+                test('can read', async () => {
+                    const client = await makeClientWithNewRegisteredAndLoggedInUser()
+                    const [organization] = await createTestOrganization(admin)
+
+                    const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                        canManageMobileFeatureConfigs: true,
+                    })
+                    await createTestOrganizationEmployee(admin, organization, client.user, role)
+
+                    const [objCreated] = await createTestMobileFeatureConfig(admin, organization)
+
+                    const objs = await MobileFeatureConfig.getAll(client, {}, { sortBy: ['updatedAt_DESC'] })
+
+                    expect(objs).toHaveLength(1)
+                    expect(objs[0]).toMatchObject({
+                        id: objCreated.id,
+                    })
+                })
+            })
+        })
+
+        describe('Validation tests', () => {
+
+            test('organization uniq constraint', async () => {
+                const [organization] = await createTestOrganization(admin)
+
+                await createTestMobileFeatureConfig(admin, organization, {
+                    commonPhone: undefined,
+                    ticketSubmittingIsDisabled: false,
+                })
+
+                await catchErrorFrom(async () => {
```

Comment:
`expectToThrowGQLError`?

Reply:
It seems that this is not a `GQLError`, this is an error that the database throws. The `constraint` error is most likely the only way to catch

Reply:
you can use `expectToThrowUniqueConstraintViolationError`